scripts/analyze

```diff
@@ -0,0 +1,168 @@
+#!/usr/bin/env python3
+
+import os
+import sys
+import json
+import re
+import shutil
+
+force = ['']
+featurefile = ''
+
+for i in sys.argv:
+    if i == sys.argv[0]:
+        continue
+    option = i.split('=')[0]
+    value = i.split('=')[1]
+    if option == 'BOARD':
+        board = value
+    elif option == 'SCRIPT':
+        script = value
+    elif option == 'FORCE':
+        force = value.split(',')
+    elif option == 'PRJBASE':
+        prjbase = value
+    elif option == 'PRJCONF':
+        prjconf = value
+    elif option == 'MAKEFILE':
+        makefile = value
+    elif option == 'MAKEBASE':
+        makebase = value
+    elif option == 'PROFILE':
+        featurefile = value
```

Comment:
Not required, but this could all be simplified with something like:

```python
args = {}   # also, set any defaults here
...
    option, value = ...
    args[option] = value
```

Then later refer to:
args['BOARD'] etc.